prereise/gather/demanddata/transportation_electrification/const.py

```diff
@@ -1,5 +1,53 @@
 model_year = 2040
 
+ldv_location_allowed = {
+    1: {1},  # home only
+    2: set(range(1, 13)),  # home and go to work related
+    4: {1, 21},  # home and school only
+    5: {1, 11, 12, 21},  # home and work and school
+    6: {10, 11, 12},  # only work
+}
+
+hdv_location_allowed = {
+    1: {1},  # base only
+}
+
+why_to_list = [
```

Comment:
Waht does `why` stand for?

Reply:
`why_to` is a field from the NHTS dataset.  Repeating the link from another comment, the NHTS has `Purpose Codes` that designate `why` a driver was driving `to` a particular place.  The full list is on pages 38-39
https://nhts.ornl.gov/assets/NHTS2017_UsersGuide_04232019_1.pdf

Reply:
Thanks for the explanation. I find the name of the variable confusing since it seems that we convert why to a list. Why not just using why_to?

Reply:
Fair point - it's a `list` of the `why_to` values, as opposed to converting `why` to a list... we were discussing with Kate this morning some ideas for changing some of the confusing variable names, with `why_to` as one of them.  Nothing's decided yet, but we'll keep you posted.